src/hooks/useDrafts/useDrafts.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import { addDraft, clearDrafts, getDraft, getDrafts, removeDraft, updateDraft } from './utils'
+
+export type CommonDraftProps = {
+  id: string
+  updatedAt?: string
+}
+
+export type DraftType = 'videos' | 'channels'
+
+export type VideoDraft = {
+  type: 'videos'
+  title?: string
+  description?: string
+  publicness?: 'public' | 'private'
+  publishedBefore?: string
+  marketing?: boolean
+  contentRating?: 'all' | 'mature'
```

Comment:
We should use the same field names as in the schema. So:
- `publicness` -> `isPublic`
- `marketing` -> `hasMarketing`
- `contentRating` -> `isExplicit`
- `publishedBefore` -> `publishedBeforeJoystream`